modules/enable-monitoring.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: PROCEDURE
+[id="enable-monitoring_{context}"]
+= Enabling monitoring
+
+Before you can monitor {product-title} you must enable monitoring.
+
+.Procedure
+. Patch the services to expose the port number `9090`.
+.. Patch the Sensor service:
+* On {ocp}:
++
+[source,terminal]
+----
+$ oc -n stackrox patch svc/sensor -p '{"spec":{"ports":[{"name":"monitoring","port":9090,"protocol":"TCP","targetPort":9090}]}}'
+----
+* On Kubernetes:
```

Comment:
Is there a reason that you show the kubernetes version for this step but not the below on patching central?

Reply:
The Sensor can run on Both Kubernetes and OCP, however, Central can only run on OCP (legally). So this is fine.